client/src/components/geneExpression/quickGene.tsx

```diff
@@ -69,7 +71,18 @@ function QuickGene() {
           }
 
           setStatus("success");
-          setGeneNames(df.col(varIndex).asArray() as DataframeValue[]);
+
+          setGeneNames(
+            df
+              .col(varIndex)
+              .asArray()
+              .filter((_, index) => {
+                const isFilteredValue = isFiltered.col(isFilteredCol).asArray()[
```

Comment:
i doubt this materializes the full array in memory on every iteration of the loop as these chained functions are probably accessors, but just to be safe, can we save the isFiltered array outside the loop to a constant?